crates/swc_cli/build.rs

```diff
@@ -0,0 +1,41 @@
+use std::{
+    env,
+    fs::{self, File},
+    io::{BufWriter, Write},
+    path::{Path, PathBuf},
+};
+
+use toml::Value;
+
+/// Creates a compile time constant to pick up latest known working version of
+/// swc_core for the plugin template.
+/// This'll ensure each version of published swc_cli will create a plugin
+/// template can be compiled correctly, then user can try to upgrade dependency
+/// as needed.
+fn main() {
+    let core_path = PathBuf::from(&env::var("CARGO_MANIFEST_DIR").unwrap())
```

Comment:
We don't need to parse `toml` file.
https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-crates

Reply:
It returns version of `self`, isn't it? for example, if I try `env::var("CARGO_PKG_VERSION")` it returns 0.89.0, instead of 0.5.0 for `swc_core`. The thing I'm trying to do is ref _another_ pkg (swc_core) to embed its version into template.

Reply:
Ah I see

Reply:
I think `swc_core` should export it instead
It would allow sharing it from various bindings

Reply:
huh, yeah, good point. Let me revise changes here.

Reply:
Ok, updated to let swc_core exposes version by self.